packages/common/src/types/scheduler.ts

```diff
@@ -234,3 +236,10 @@ export type ApiCsvUrlResponse = {
         status: string;
     };
 };
+
+export type CompileProjectPayload = {
+    userUuid: string;
+    projectUuid: string;
+    requestMethod: string;
```

Comment:
There is a `RequestMethod` type we can use here I think (unless graphile doesn't like that)

Reply:
Yes, I use the method to parse it into a string on the http request, but his is going to write it into a JSON blob in database so I prefer this to be a string when it happens.